File: Java/LeapYear.java
package Java;
import java.util.Scanner;

public class LeapYear {

	public static void main(String args[])
	{
		System.out.print("Enter the Year: ");
		Scanner sc = new Scanner(System.in);
		int year = sc.nextInt();
		
		if(((year%100!=0 & year%4==0 )) | year%400==0)
		{

			System.out.print("Entered Year is a Leap year");
		}
		else
		{
			System.out.print("Entered Year is not a Leap year");
		}
	}
	
}

File: Python/LeapYear.py
# Leap Year

year = int(input("Enter the Year: "))

flag = False

if(year%4==0):
	if(year%100==0):
		if(year%400==0):
			flag=True
		else:
			flag = False
	else:
		flag = True
else:
	flag = False

if(flag is True):
	print("Entered year is a Leap Year")
else:
	print("Entered year is not a Leap Year")

